modules/appeals_api/app/services/appeals_api/pdf_construction/generator.rb

```diff
@@ -7,6 +7,7 @@ module PdfConstruction
     class Generator
       def initialize(appeal, version: 'V1')
         @appeal = appeal
+        appeal.update(pdf_fill_version: version)
```

Comment:
Maybe I'm missing something here, but why are we updating the appeal itself in the pdf generator?

Reply:
This version is a direct reference to the version of pdf that we fill for that specific appeal. This is where the generator utilizes the version number to apply the correct structure (pdf version) to the pdf generation. So this is where we store that same version, at the time of generating the PDF.

Reply:
We could (in theory) do it elsewhere, but this kind of circumvents the edge case of us using a different PDF version from the API version.